tests/integration/clarity/natives/index-of.spec.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  makeContractDeploy,
+  broadcastTransaction,
+  AnchorMode,
+  PostConditionMode,
+  TxBroadcastResultOk,
+  makeContractCall,
+  SignedContractCallOptions,
+} from "@stacks/transactions";
+import { StacksNetwork, StacksTestnet } from "@stacks/network";
+import { Accounts, Constants } from "../../constants";
+import {
+  buildDevnetNetworkOrchestrator,
+  waitForStacksChainUpdate,
+  waitForStacksTransaction,
+} from "../../helpers";
+import { DevnetNetworkOrchestrator } from "@hirosystems/stacks-devnet-js";
+
+describe("index-of?", () => {
+  let orchestrator: DevnetNetworkOrchestrator;
+  let network: StacksNetwork;
+
+  beforeAll(() => {
+    orchestrator = buildDevnetNetworkOrchestrator();
+    orchestrator.start();
+    network = new StacksTestnet({ url: orchestrator.getStacksNodeUrl() });
+  });
+
+  afterAll(() => {
+    orchestrator.stop();
+  });
+
+  const codeBody = `(define-public (test-1)
+    (ok (index-of? u"hiro" u"i"))
+)
+(define-public (test-2)
+    (ok (index-of? 0x00112233 0x22))
+)
+(define-public (test-3)
+    (ok (index-of? (list 1 2 3 4) 5))
+)`;
```

Comment:
should we add a test for `ascii-string` sequences?

Reply:
The intent of these tests was not to cover all paths, but mostly just to make sure that these new functions work in 2.1 and not in 2.05. The coverage of all cases should be in the unit tests. I just covered a few different cases in each one. Let me know if you think full coverage should be included here.